integration_tests/run_pyspark_from_build.sh

```diff
@@ -76,7 +76,7 @@ else
         # Make sure we have Parquet version >= 1.12 in the dependency
         LOWEST_PARQUET_JAR=$(echo -e "$MIN_PARQUET_JAR\n$PARQUET_HADOOP_TESTS" | sort -V | head -1)
         export INCLUDE_PARQUET_HADOOP_TEST_JAR=$([[ "$LOWEST_PARQUET_JAR" == "$MIN_PARQUET_JAR" ]] && echo true || echo false)
-        PLUGIN_JARS=$(echo "$TARGET_DIR"/../../dist/target/rapids-4-spark_*.jar)
+        PLUGIN_JARS=$(echo "$TARGET_DIR"/../../dist/rapids-4-spark_*.jar)
```

Comment:
unrelated, If this is a real issue let us fix in a separate PR

Reply:
Hmm, thanks for catching this! It must be a mistake. I didn't intend to modify this line. Reverted it.